src/NovelRT/Graphics/ImageRect.cpp

```diff
@@ -123,17 +123,11 @@ namespace NovelRT::Graphics {
         auto cFile = fopen(_imageDir.c_str(), "rb");
         auto png = png_create_read_struct(PNG_LIBPNG_VER_STRING, nullptr, nullptr, nullptr); //TODO: Figure out how the error function ptr works
 
-        if (png == nullptr) {
-          _logger.logErrorLine("Image file cannot be opened! Please ensure the path is correct and that the file is not locked.");
-          throw std::runtime_error("Unable to continue! File failed to load for texture.");
-        }
+        _logger.throwIfNullPtr(png, "Image file cannot be opened! Please ensure the path is correct and that the file is not locked.");
 
         auto info = png_create_info_struct(png);
 
-        if (info == nullptr) {
-          _logger.logError("Image at path " + _imageDir + " failed to provide an info struct! Aborting...");
-          throw std::runtime_error("Unable to continue! File failed to load for texture.");
-        }
+        _logger.throwIfNullPtr(info, "Image at path " + _imageDir + " failed to provide an info struct! Aborting...");
```

Comment:
@tannergooding based on how we wanted to handle multiple strings,  should the function be changed to support it (a la the `logError` template)? Not sure if we want to stray from  it for this PR or not.

Reply:
This should have been resolved with <https://github.com/novelrt/NovelRT/pull/141>